diagnostic.md

```diff
@@ -4,43 +4,43 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1.We just forked a repo on GitHub and want to start working on it locally.
 What command do we use to do that? Use the **correct** URL from your fork of
-this repository in your answer.
+this repository in your answer .
 
-```sh
-<!-- Remove this comment and place your answer here. -->
+```
+git clone https://github.com/ga-wdi-boston/git-basics-diagnostic.git
```

Comment:
Should be `git clone git@github.com:mattb5/git-basics-diagnostic.git`. This clones the SSH from **your** forked repo.